cli/Dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM alpine:3.5
 MAINTAINER Kontena, Inc. <info@kontena.io>
 
-ADD https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 /usr/local/bin/docker
+ADD https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz /tmp/docker-latest.tgz
+RUN tar --strip-components=1 -xvzf /tmp/docker-latest.tgz -C /usr/local/bin
+RUN rm /tmp/docker-latest.tgz
```

Comment:
This still leaves the `/tmp/docker-latest.tgz` in the image, because each command is a separate layer. Use something like this instead?

```
RUN curl -sL https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz | tar -C /usr/local/bin --strip-components=1 -xzv
```